test/tests.js

```diff
@@ -1,14 +1,233 @@
 // IMPORT MODULES under test here:
-import { myFunction } from '../functions.js';
+import {
+    myFunction,
+    addExclamationPoints,
+    multiplyBy12ThenHalve,
+    multiplyBySeven,
+    divideThenMultiply,
+    returnAsAnArray,
+    returnAsAString,
+    makeLuckyGreeting,
+    getSecondItem,
+    getLastItem,
+} from '../functions.js';
 
 const { test, skip } = QUnit;
 
-test('this test should pass', (expect) => {
-    const expected = true;
+test('this test should add "!!!" at the end of a string or number', (expect) => {
+    const expected = 'this is cool' + '!!!';
+
+    const actual = addExclamationPoints('this is cool');
+
+    expect.equal(actual, expected, 'this is cool' + '!!!');
+});
+
+test('this test should add "!!!" at the end of a string or number', (expect) => {
+    const expected = 'this is something else' + '!!!';
+
+    const actual = addExclamationPoints('this is something else');
+
+    expect.equal(actual, expected, 'this is something else' + '!!!');
+});
+test('this test should add "!!!" at the end of a string or number', (expect) => {
```

Comment:
you are welcome to write a new test for each actual/expected, but you can also add multiple test cases into a single test like this: 

```
test('this test should add "!!!" at the end of a string or number', (expect) => {
    const expected = 'this is cool' + '!!!';
    const expected2 = 'this is something else' + '!!!';

    const actual = addExclamationPoints('this is something else');
    const actual2 = addExclamationPoints('this is cool');

    expect.equal(actual, expected, 'this is cool' + '!!!');
});
expect.equal(actual2, expected2, 'this is cool' + '!!!');
});
``` etc. etc. etc.